README.md

```diff
@@ -1,6 +1,6 @@
 # How to Git Better at Git
 
-- Daric Teske
+- Allison Inman didn't write this, Daric Teske did
```

Comment:
You forgot a period at the end of your sentence, pls change otherwise Amy Cyphers will be mad.

Reply:
Plz don't tell her I am so scared